go.mod

```diff
@@ -3,27 +3,29 @@ module github.com/rotationalio/watermill-ensign
 go 1.19
 
 require (
-	github.com/ThreeDotsLabs/watermill v1.1.1
+	github.com/ThreeDotsLabs/watermill v1.2.0
 	github.com/google/uuid v1.3.0
-	github.com/pkg/errors v0.8.1
-	github.com/rotationalio/go-ensign v0.4.1
+	github.com/pkg/errors v0.9.1
+	github.com/rotationalio/go-ensign v0.5.2-0.20230329215109-1db6c9a1ec1c
```

Comment:
Note that when we tag `github.com/rotationalio/go-ensign` -- we'll want to update this; right now it's pulling directly from a specific commit.